LocationUpdatesBackgroundKotlin/app/src/main/java/com/google/android/gms/location/sample/locationupdatesbackgroundkotlin/data/db/MyLocationEntity.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.android.gms.location.sample.locationupdatesbackgroundkotlin.data.db
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+import java.text.DateFormat
+import java.util.Date
+import java.util.UUID
+
+/**
+ * Data class for Location related data (only takes what's needed from
+ * {@link android.location.Location} class).
+ */
+@Entity(tableName = "my_location_table")
+data class MyLocationEntity (@PrimaryKey val id:UUID = UUID.randomUUID(),
```

Comment:
Could be called `LocationEntity`
Put the `id` on the next line and re-format the file

Reply:
Done.